### PR TITLE
feat: add examples for working with `File.flix`

### PR DESCRIPTION
**Pull request by ninaandrup**
Related to #2807

### CODE REVIEW

**Comment by ninaandrup** [comment]
@magnus-madsen Did you want more than one example?

I could maybe split this into two, where one is about getting information of file, and one is reading and stuff.

**Comment by magnus-madsen** [comment]
Lets do two examples. Let us also ensure they fit in the box on www.flix.dev.

**Review by magnus-madsen — COMMENTED** [review]
I like both examples. Do they fit in the box on flix.dev in terms of width?

**Comment by ninaandrup** [comment]
> I like both examples. Do they fit in the box on flix.dev in terms of width?

I'm not sure about that. I will check

File: examples/file-information.flix [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// Getting information on files with Flix.
+def main(_args: Array[String]): Int32 & Impure =
+  let f = "README.md";
+
+  // Check if the file 'README.md' exists.
+  match File.exists(f) {
+    case Ok(exist) => {
+      println("The file ${f} exists: ${exist}.")
+    }
+    case Err(msg)  => println("An error occurred with message: ${msg}")
```
Comment (ninaandrup) [inline examples/file-information.flix:10]: I'm not sure if this is propper English. Can you say `an error occurred with message`??

**Comment by ninaandrup** [comment]
> > I like both examples. Do they fit in the box on flix.dev in terms of width?
> 
> I'm not sure about that. I will check

I'm pretty sure they fit in the box on the page now :)

File: examples/file-information.flix (hunk 1 above)
Comment (magnus-madsen) [reply examples/file-information.flix:10]: Think it is OK.

**Comment by magnus-madsen** [comment]
Thanks!

**Pull request merged**